cf-agent/Makefile.am

```diff
@@ -36,6 +37,7 @@ AM_CPPFLAGS = -I$(srcdir)/../libpromises -I$(srcdir)/../libntech/libutils \
 	$(PAM_CPPFLAGS)
 
 AM_CFLAGS = \
+	@CFLAGS@ \
```

Comment:
Shouldn't this come last to make sure it overrides potentially conflicting flags from the other variables?

Reply:
I added it here to match how it's done in cf-promises. Note that this is the CFLAGS from autogen / configure, not command line arg. Yes, it could probably be in the end (ensuring that flags we set are added after flags from dependencies), however I don't think this is a big deal. I try to make incremental changes to fix the immediate problem.